packages/search/src/ago/helpers/filters/has-api.ts

```diff
@@ -0,0 +1,21 @@
+import { getProp } from "@esri/hub-common";
+const apiTypes = ["Feature Service", "Map Service", "Image Service"];
+
+export function hasApi(queryFilters: any) {
```

Comment:
when this library is bundled as a UMD all the methods hang off of an `arcgisHub` namespace. 

this means that in the browser, `filters.hasApi` will collide with `aggs.hasApi`

Reply:
Interesting... the UI is not supposed to calls these functions directly. They are being used by `agoSearch` and `computeItemsFacets` that are exposed by this library and the UI will talk to just those functions. Do I have still rename them?

Reply:
i didn't see before that you included `"hasApi"` in the blacklist. thats definitely good.

even if callers that load the UMD don't call these methods directly though, things will go pear shaped when they're called _internally_, because i don't think there is a mechanism in place in that build to differentiate them.

you have two options:
1. make the names unique, just to be safe.
2. test loading the UMD in the browser and calling both methods that lean on `hasApi` internally to confirm that i'm wrong.